### DISCUSSION TITLE
Turborepo Build Not Including global.css

### DISCUSSION THREAD

**Discussion by jrock2004** [vercel/turborepo#4270] [Help] [answered]
So run I run development, all my apps are using the global css file just fine. But when I build my apps and serve up the `server/pages` directory, the CSS files are not there. Thoughts?

Is it because the CSS files are in static? Maybe I need to do more then the build?

**Comment by BernieceMarinaroKlO** [comment]
It's possible that the CSS files are not being included in your build because they are in the static directory, which is not automatically included in the build process. You may need to configure your build process to include the static directory and its contents.

Alternatively, it could be a problem with the path to the CSS files in your build. Make sure that the paths are correct and that the CSS files are being referenced correctly in your HTML files.

If you're still having trouble, you may need to provide more information about your build process and file structure for further diagnosis.

**Reply by jrock2004** [reply, marked as answer]
I noticed that in the build folder there is another folder that is not a sibling that has the CSS, so I am guessing you are right and I have them in the static dir

**Comment by anthonyshew** [comment]
We'll need more information to be able to help here. A reproduction would be great.
